truffle/src/com.oracle.truffle.host/src/com/oracle/truffle/host/HostExecuteNode.java

```diff
@@ -596,7 +595,7 @@ private SingleMethod findBestCandidate(List<SingleMethod> applicableByArity, Obj
                     for (int i = 0; i < paramCount; i++) {
                         if (!HostToTypeNode.canConvert(args[i], parameterTypes[i], genericParameterTypes[i], null,
                                         hostContext, priority, InteropLibrary.getFactory().getUncached(args[i]),
-                                        HostTargetMappingNode.getUncached())) {
+                                        HostTargetMappingNode.getUncached(), HostLibrary.getUncached())) {
```

Comment:
OK to use `HostLibrary.getUncached()` as there already was `HostTargetMappingNode.getUncached()`, I think.